README.md

```diff
@@ -183,12 +183,15 @@ Sent from Mission Control to instruct the rover server to begin providing a came
 ```
 {
   type: "cameraStreamOpenRequest",
-  camera: string
+  camera: string,
+  fps: int
```

Comment:
`int` isn't a thing in JS, can you change it to `number` and then specify in the docs that it should be an integer?